### DISCUSSION TITLE
Per-user pricing for non-technical Project Manager users who only manage issues and projects

### DISCUSSION THREAD

**Discussion by devth** [community/community#122588] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

In the [per-user pricing docs](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/about-per-user-pricing) it mentions:

> GitHub does not bill for the following people:
> Billing managers

Have ya'll considered a similar policy and role for non-technical PM users who do not contribute code or use any of the other features? We are considering consolidating our Project Management from Asana over to GitHub now that Projects appears to be quite powerful, but I wouldn't want to pay for a full license for every non-technical user. We only recently upgraded to a paid GitHub Team plan. This pricing structure seems to incentivize downgrading to the Free tier. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by chasegiunta** [comment]
Same, stuck with Jira because managers, etc who don't touch code would cost per user in GitHub.
